### PR TITLE
Update README to document sum and mean aggregations

### PR DESCRIPTION
**Pull request by songy23**
Fixes https://github.com/census-instrumentation/opencensus-go/issues/232

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,14 +136,18 @@ if err := stats.DeleteMeasure(m); err != nil {
 
 ### Creating an aggregation
 
-Currently only 2 types of aggregations are supported. The CountAggregation is used to count
+Currently 4 types of aggregations are supported. The CountAggregation is used to count
 the number of times a sample was recorded. The DistributionAggregation is used to
-provide a histogram of the values of the samples.
+provide a histogram of the values of the samples. The SumAggregation is used to
+sum up all sample values. The MeanAggregation is used to calculate the mean of
+smaple values.
```
Comment (odeke-em) [inline README.md]: sample

File: README.md [hunk 2]
```diff
@@ -136,14 +136,18 @@ if err := stats.DeleteMeasure(m); err != nil {
 
 ### Creating an aggregation
 
-Currently only 2 types of aggregations are supported. The CountAggregation is used to count
+Currently 4 types of aggregations are supported. The CountAggregation is used to count
```
Comment (rakyll) [inline README.md]: Can you update the source here at https://github.com/census-instrumentation/opencensus-go/blob/master/internal/readme/source.md?

README is autogenerated by running the script, see https://github.com/census-instrumentation/opencensus-go/tree/master/internal/readme.

Sorry, this should be more obvious but I didn't have time yet to make it more visible.

Comment (songy23) [reply README.md]: No problem, updated.

File: README.md (hunk 1 above)
Comment (songy23) [reply README.md]: Done.

**Comment by rakyll** [comment]
Can you also regenerate the README? Instructions are at https://github.com/census-instrumentation/opencensus-go/tree/master/internal/readme.

**Comment by songy23** [comment]
Sure, I updated README with embedme.

**Comment by songy23** [comment]
Thanks!

**Comment by odeke-em** [comment]
@rakyll how come we have to generate the README in the root, from the internal README? Am just curious.

**Comment by rakyll** [comment]
I will write a CONTRIBUTING.md soon and make it more obvious.

**Pull request merged**